src/camera_model.cpp

```diff
@@ -39,3 +44,12 @@ CameraModel stringToModel(const std::string &model) {
     else
         return Unknown;
 }
+
+bool isGrayScale(CameraModel model) {
+    if (std::find(std::begin(grayScaleCameras), std::end(grayScaleCameras),
```

Comment:
Should check if the found pointer is equal to the end of the vector.  Also, simplyify -- you don't need if (thing) return true, just return (thing)